### PR TITLE
docs - update when pg_class.relfrozenxid is 0 (zero)

### PR DESCRIPTION
**Pull request by mkiyama**
This will be backported to 5X_STABLE

### CODE REVIEW

**Comment by mkiyama** [comment]
@jimmyyih 
This is the PR for relfrozenxid. 
Thanks for your help

**Comment by jeff8569** [comment]
I did not take a part of the crap

File: gpdb-doc/dita/admin_guide/intro/mvcc_example.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
    <body>
       <p>For Greenplum Database, the transaction ID (XID) value an incrementing 32-bit
             (2<sup>32</sup>) value. The maximum unsigned 32-bit value is 4,294,967,295, or about
-         four billion. The XID values restart at 0 after the maximum is reached. Greenplum Database
+         four billion. The XID values restart at 1 after the maximum is reached. Greenplum Database
```
Comment (jimmyyih) [inline gpdb-doc/dita/admin_guide/intro/mvcc_example.xml]: I think this should be 3, not 1.

Comment (mkiyama) [reply gpdb-doc/dita/admin_guide/intro/mvcc_example.xml]: @jimmyyih 
Just curious, what are values 1 and 2 used for?

Comment (mkiyama) [reply gpdb-doc/dita/admin_guide/intro/mvcc_example.xml]: Thanks Jimmy.

**Pull request merged**